scipy-1.0/paper.tex

```diff
@@ -151,6 +151,34 @@ \subsection*{Numerical optimization}
 
 
 \subsection*{Statistical distributions}
+Version 1.0 of SciPy saw the addition of the energy distance algorithm in
+\texttt{scipy.stats.energy\_distance}. Energy distance is used in a varity of applications ranging
+from genetic analysis to multivariate independence. We also added the
+first Wasserstein distance algorithm to the library in
+\texttt{scipy.stats.wasserstein\_distance}. The Wasserstein distance is often
+used in color digital image analysis.
+
+We now also provide the cumulative distribution function of the multivariate
+normal distribution, \\\texttt{scipy.stats.multivariate\_normal}, via the newly
+associated \texttt{logcdf} and \texttt{cdf} methods.
+
+Other recent additions to \texttt{scipy.stats} include the Argus distribution
+in \texttt{scipy.stats.argus} (used in particle physics), a class for
+generating a distribution from a histogram in
+\texttt{scipy.stats.rv\_histogram}, an implementation of the multinomial
+distribution in \texttt{scipy.stats.multinomial} (used for example in natural
+language processing), a weighted Kendall's tau statistic algorithm in
+\texttt{scipy.stats.weightedtau}, interquantile range handling in
+\texttt{scipy.stats.iqr}, the Burr type XII distribution is available in
```

Comment:
Everything before Burr is written like "____ in _____", then the structure starts varying like "_____ is available in _____" and "_____ has been implemented as _____". I would suggest staying consistent throughout or changing every time.